resolwe_bio/processes/reads_processing/bbduk.yml

```diff
@@ -11,221 +11,363 @@
     expression-engine: jinja
     executor:
       docker:
-        image: resolwebio/legacy:1.0.0
-  data_name: '{{ reads.fastq.0.file|default("?") }}'
-  version: 0.0.6
+        image: resolwebio/rnaseq:3.0.0
+    resources:
+      cores: 10
+  data_name: '{{ reads|sample_name|default("?") }}'
+  version: 0.0.7
   type: data:reads:fastq:single:bbduk
   category: analyses
   flow_collection: sample
   persistence: CACHED
   description: >
-    BBduk is capable of quality-trimming and filtering, adapter-trimming, contaminant-filtering via
-    kmer matching, sequence masking, GC-filtering, length filtering, entropy-filtering, format
-    conversion, histogram generation, subsampling, quality-score recalibration, kmer cardinality
-    estimation, and various other operations in a single pass.
+    BBDuk combines the most common data-quality-related trimming, filtering, and masking operations
+    into a single high-performance tool. It is capable of quality-trimming and filtering,
+    adapter-trimming, contaminant-filtering via kmer matching, sequence masking, GC-filtering,
```

Comment:
Isn't it "k-mer"? For also other occurrencies.

Reply:
I used the same spelling as in [the official documentation](https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbduk-guide/). [Wikipedia](https://en.wikipedia.org/wiki/K-mer) uses your suggestion though.

Reply:
True, leave it as it is. I guess it's a matter of preference...